### PR TITLE
adding '-a' option to get_all_images

### PR DESCRIPTION
**Pull request by raskin-aleksandr**

### CODE REVIEW

**Comment by meirwah** [comment]
where is the documentation change for that?

File: content/io/cloudslang/docker/containers/get_all_containers.sl [hunk 1]
```diff
@@ -34,6 +34,13 @@ imports:
 flow:
   name: get_all_containers
   inputs:
+    - all_containers:
+        default: False
+    - ps_params:
+        default: " '-a' if bool(all_containers) else ''"
```
Comment (orius123) [inline content/io/cloudslang/docker/containers/get_all_containers.sl]: you can do it in oneline.
`ps_params: " '-a' if bool(all_containers) else ''"`

**Comment by meirwah** [comment]
:+1:

**Comment by meirwah** [comment]
We want to merge this right?

**Comment by meirwah** [comment]
:+1:

**Pull request merged**